core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -255,12 +255,17 @@ private[spark] object Utils extends Logging {
    * @return true if the permissions were successfully changed, false otherwise.
```

Comment:
@MartinWeindel, mind adding a comment to this Javadoc to explain that this method is a no-op under Windows?